src/components/Calendar/doubleCalendar/yearSelect/styled/index.js

```diff
@@ -27,6 +27,23 @@ export const StyledContainer = attachThemeAttrs(styled.div)`
         `};
 `;
 
+export const StyledLabel = styled.label`
+    position: absolute !important;
+    margin: -1px !important;
+    border: 0 !important;
+    padding: 0 !important;
+    width: 1px !important;
+    height: 1px !important;
+    overflow: hidden !important;
+    clip: rect(0 0 0 0) !important;
+    text-transform: none !important;
+    white-space: nowrap !important;
+
+    &:empty {
+        margin: 0;
+    }
+`;
```

Comment:
in order to avoid DRY (don't repeat yourself) we have a hidden element:
```src/components/Structural/hiddenElement.js```
that you can use as:
```<HiddenElement as="label" />```